### DISCUSSION TITLE
Post title showing page title on homepage

### DISCUSSION THREAD

**Discussion by LikeOMG** [GetPublii/Publii#1607] [Help]
I hope this makes some kind of sense! My {{title}} tag produces the *website name* instead of the post name, and when that error started, none of the posts would show up. Just today's date beneath the website title. I did a couple of different searches in hopes of finding an answer, but I'm new to this so I might not have searched for the right thing. Any help appreciated!

EDIT: Decided not to code from scratch.

**Comment by bobmitro** [comment]
There is a lack of context. What page are you talking about: post, tag, etc...?

**Reply by LikeOMG** [reply]
As stated above, this is the homepage, and the website/page title was showing where the **_post_** titles should be. It's fine, though, I decided to edit a template instead of starting from scratch until I'm used to the system. Thanks, anyway!
